### DISCUSSION TITLE
routine for service callbooks

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#449] [Ideas]
Hello again, I can't stop finding things to do with this software.

My daily routine is, at the end of the radio session, to upload the logs to eqsl, lotw, hdrlog... and download the new ones.

It's almost a ritual...but I thought someone must find it tedious.
Maybe you could make a routine that does all this automatically.
We could tell the program who we want to interact with and in what order.
I personally start with LotW, download eQsl, then HdrLog, and finally Clublog (which I have configured to send live qsl) but this way I update clublog with eQSL and LotW downloads.
I don't use QRZ because I don't understand that I have to pay to send the log via API.
It's an idea, to put at the end of the list.

**Comment by foldynl** [comment]
I received an idea through the mailing list to unify upload and download from/to all services into one dialog. I really like that idea. I opened an issue #448 for it.
